WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -63,6 +71,28 @@ public void onCreate(Bundle savedInstanceState) {
             // first time it's created
             mTopPagerSelectedButtonIndex = getArguments().getInt(ARGS_TOP_PAGER_SELECTED_BUTTON_INDEX, 0);
         }
+
+        // Single background thread used to create the blogs list in BG
+        ThreadPoolExecutor blogsListCreatorExecutor = (ThreadPoolExecutor) Executors.newFixedThreadPool(1);
+        blogsListCreatorExecutor.submit(new Thread() {
+            @Override
+            public void run() {
+                // Read all the dotcomBlog blogs and get the list of home URLs.
+                // This will be used later to check if the user is a member of followers blog marked as private.
+                List <Map<String, Object>> dotComUserBlogs = WordPress.wpDB.getAccountsBy("dotcomFlag=1", new String[]{"homeURL"});
+                for (Map<String, Object> blog : dotComUserBlogs) {
+                    if (blog != null && blog.get("homeURL") != null) {
+                        String normURL = UrlUtils.normalizeUrl(blog.get("homeURL").toString()).toLowerCase();
```

Comment:
It seems error-prone to me to duplicate this same URL formatting code [here](https://github.com/wordpress-mobile/WordPress-Android/pull/2180/files#diff-7f97a9a7e8ae77d48b26a20bdb986d46R260) since future devs may not be careful enough to update both of them if they need changing. How about using a separate routine to provide the formatting, like this:

```
private static String normalizeAndRemoveScheme(String url) {
    if (TextUtils.isEmpty(url)) {
        return "";
    }
    String normURL = UrlUtils.normalizeUrl(url.toLowerCase());
    int pos = normURL.indexOf("://");
    if (pos > -1) {
        return normURL.substring(pos + 3);
    } else {
        return normURL;
    }
}
```